views-react/build.gradle

```diff
@@ -0,0 +1,86 @@
+plugins {
+    id "io.micronaut.build.internal.views-module"
+    id "com.github.node-gradle.node" version "7.0.2"
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
+    annotationProcessor(mnValidation.micronaut.validation.processor)
+
+    api projects.micronautViewsCore
+    api libs.jetbrains.annotations
+    implementation(mn.micronaut.http.client)
+
+    // The user of this library is expected to supply the JS dependency. That's because they may choose between
+    // the community edition (open source, slower) and the enterprise edition (liberal usage, not open source, faster).
+    implementation(libs.graal.polyglot)
+
+    compileOnly(mn.micronaut.management)
+    compileOnly(mnValidation.micronaut.validation)
+    compileOnly(mn.micronaut.http)
```

Comment:
why are these compileOnly?

Reply:
I fixed two, left validation because it's only used for `@NotBlank`.